bin/glrd-manage

```diff
@@ -0,0 +1 @@
+glrd
```

Comment:
what's the use-case for this script?

Reply:
`bin/glrd-manage` is just a symlink to `bin/glrd`. It has some magic in it ...
https://github.com/gardenlinux/gardenlinux/pull/2409/files#diff-ff3a914bba97885ac56a104c745a59a74bae8df634b5cf25396dc029f17e98bfR2
https://github.com/gardenlinux/gardenlinux/pull/2409/files#diff-ff3a914bba97885ac56a104c745a59a74bae8df634b5cf25396dc029f17e98bfR15
... which enables us to use the same code (as in the link target) without having a very similar file twice.
It basically calls the `${cmd}` in the container which is gotten from the filename/basename of `bin/glrd-manage` OR `bin/glrd`.